addons/mass_mailing_sms/models/sms_tracker.py

```diff
@@ -17,8 +17,8 @@ class SmsTracker(models.Model):
 
     mailing_trace_id = fields.Many2one('mailing.trace', ondelete='cascade')
 
-    def _action_update_from_provider_error(self, provider_error):
-        error_status, failure_type, failure_reason = super()._action_update_from_provider_error(provider_error)
+    def _action_update_from_provider_error(self, provider_error, failure_reason=None):
```

Comment:
You can't do that in stable: it means any custom addon overriding the method is broken as it is going to receive an unexpected value.

Reply:
Ah right..
Aside from a context value (:hankey:), I guess you'll need to update notification/trace manually or in a wrapper around this (also kind of :hankey: to be honest).

Reply:
Sorry about that one...

I think I'd go with the context in stable, so the cleanup is straightforward in master.